.travis.yml

```diff
@@ -3,19 +3,30 @@ dist: trusty
 
 language: generic
 
+env:
```

Comment:
Sorry to state the obvious but try without that line?  Or can matrix go inside the `env:`?

Otherwise, I guess we set `DOCTEST=no` for these and leave a comment about `allow_failures` for now?